cmd/utils.go

```diff
@@ -1060,6 +1060,13 @@ func getStackLabels(config *RootCommandConfig) (map[string]string, error) {
 		}
 	}
 
+	imageAndDigest := data[0]["RepoDigests"].([]interface{})
```

Comment:
I think we should be looking at the `id` field given my the inspect command. I build a stack by running a `stack package` and my `RepoDigests` was empty.

Reply:
I suppose this makes more sense if we're looking at uniquely identifying the images since locally built images won't generate a digest. Added in the latest commit.

Reply:
@EnriqueL8 I've just reverted that commit to add the image id instead of the digest. Spoke with @neeraj-laad and we need the digest. Since locally built images don't generate one we'll just skip adding the label for those cases.

Reply:
Makes sense, I would only add the `RepoDigests` then.